src/components/about-us/index.js

```diff
@@ -46,7 +45,10 @@ export class AboutUs extends PureComponent {
     this.sectionOffset = []
   }
   _handleClickAnchor = anchorIdx => {
-    return smoothScroll(this.sectionOffset[anchorIdx])
+    window.scroll({
+      top: this.sectionOffset[anchorIdx],
+      behavior: 'smooth',
```

Comment:
It seems to me that [Safari does not have support for the smooth scroll behavior](https://developer.mozilla.org/en-US/docs/Web/API/Window/scroll#browser_compatibility) ... 🥲

Reply:
oh no...:cry

Reply:
https://github.com/iamdustan/smoothscroll

polyfill ?

Reply:
replaced with our custom smooth scroll.